server/index.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const db = require('../db/index.js');
+const bodyParser = require('body-parser');
+// for cross-origin-request
+const cors = require('cors');
+const path = require('path');
+
+let app = express();
+app.use(bodyParser.json());
+app.use(cors());
+
+//server static content
+app.use(express.static(path.join(__dirname, '../client/dist')))
+app.get('/reviews', function(req, res) {
+    var pageIndex = Number(req.url.split('=')[1]);
+    var getFive; var len;   
```

Comment:
Wondering why you declared it up here, instead of declaring it further down when you use it

Reply:
I was doing some changes, I comited it in middle